### ISSUE TITLE
Error: Cannot find module 'request'

### ISSUE THREAD

**Issue by charlierproctor** [snlab/odl-summit-2016-tutorial-vm#2]
```
vagrant@localhost ~/tutorial/cloud9 $ ./start.sh
Starting test-config
Connect server listening at http://10.0.2.15:9000
CDN: version standalone initialized /home/vagrant/tutorial/cloud9/build
Trace: Error while starting '/home/vagrant/tutorial/cloud9/configs/test-config':
    at /home/vagrant/tutorial/cloud9/server.js:223:25
    at Architect.done (/home/vagrant/tutorial/cloud9/node_modules/architect/architect.js:587:9)
    at Architect.emit (events.js:95:17)
    at startPlugins (/home/vagrant/tutorial/cloud9/node_modules/architect/architect.js:468:21)
    at startPlugins (/home/vagrant/tutorial/cloud9/node_modules/architect/architect.js:473:21)
    at register (/home/vagrant/tutorial/cloud9/node_modules/architect/architect.js:504:13)
    at Server.<anonymous> (/home/vagrant/tutorial/cloud9/node_modules/connect-architect/connect/connect-plugin.js:135:13)
    at Server.g (events.js:180:16)
    at Server.emit (events.js:92:17)
    at net.js:1056:10
{ [Error: Cannot find module 'request']
  code: 'MODULE_NOT_FOUND',
  plugin:
   { packagePath: '/home/vagrant/tutorial/cloud9/plugins/snlab.devopen.server/managerd.js',
     provides: [ 'controller.managerd' ],
     consumes: [ 'Plugin' ],
     setup: { [Function: main] consumes: [Object], provides: [Object] } } } 'Error: Cannot find module \'request\'\n    at Module._resolveFilename (module.js:338:15)\n    at Function.modules._resolveFilename (/home/vagrant/tutorial/cloud9/node_modules/c9/setup_paths.js:26:12)\n    at Function.Module._load (module.js:280:25)\n    at Module.require (module.js:364:17)\n    at require (module.js:380:17)\n    at Object.<anonymous> (/home/vagrant/tutorial/cloud9/plugins/snlab.devopen.server/lib/test.js:1:77)\n    at Module._compile (module.js:456:26)\n    at Module.module.constructor._compile (/home/vagrant/tutorial/cloud9/node_modules/amd-loader/amd-loader.js:10:31)\n    at Object.Module._extensions..js (module.js:474:10)\n    at Module.load (module.js:356:32)'
```

**Comment by charlierproctor** [comment]
Solution: `npm install request`... should we add this to some `package.json` or to one of the bootstrap scripts?

**Comment by fno2010** [comment]
Thank you so much, @charlierproctor! Fixed it already by 310f6ae.
